pandas/core/frame.py

```diff
@@ -8149,6 +8149,11 @@ def cov(self, min_periods=None) -> "DataFrame":
             Minimum number of observations required per pair of columns
             to have a valid result.
 
+        ddof : int, default 1
+            Delta degrees of Freedom.  The divisor used in calculations
+            is ``N - ddof``, where ``N`` represents the number of elements.
+            versionadded:: 1.1.0
```

Comment:
needs a blank line before the versionadded

Reply:
done